server.js

```diff
@@ -358,59 +296,29 @@ io.on('connection', (socket) => {
                 top: topThreePolicies
             });
             socket.leave('sh-chancellor');
+            chanPolicy = data;
+            botPowers();
+            printResults();
```

Comment:
chancellor's policy stores as a variable
bot uses presidential power (if president
prints policy results to console (for debugging